### PR TITLE
week2/ssimmons_homework2

### PR DESCRIPTION
**Pull request by sydsimmons**

### CODE REVIEW

File: Week 2/ssimmons/script.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+var add = document.getElementById('addToDo');
+var input = document.getElementById('inputField');
```
Comment (bclincy) [inline Week 2/ssimmons/script.js]: Please changes these to let vs var 
```var input = document.getElementById('inputField');

 let  input = document.getElementById('inputField');
```

File: Week 2/ssimmons/script.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+var toDoContainer = document.getElementById('toDoContainer');
+
+add.addEventListener('click',addItem);
+input.addEventListener('keypress',function(e){
+    if(e.key=="Enter"){
```
Comment (bclincy) [inline Week 2/ssimmons/script.js]: if should have a space before the (

Comment (sydsimmons) [reply Week 2/ssimmons/script.js]: Fixed both of these issues

**Review by danosherow — APPROVED** [review]
looks good from what I can tell

**Pull request merged**